test/e2e/config/e2e_conf.yaml

```diff
@@ -219,9 +246,11 @@ variables:
   NODE_DRAIN_TIMEOUT: "0s"
   IRONIC_RELEASE_26.0: "data/ironic-deployment/overlays/release-26.0"
   IRONIC_RELEASE_27.0: "data/ironic-deployment/overlays/release-27.0"
+  IRONIC_RELEASE_27.0: "data/ironic-deployment/overlays/release-29.0"
   IRONIC_RELEASE_LATEST: "data/ironic-deployment/overlays/release-latest"
   BMO_RELEASE_0.8: "data/bmo-deployment/overlays/release-0.8"
   BMO_RELEASE_0.9: "data/bmo-deployment/overlays/release-0.9"
+  BMO_RELEASE_0.10: "data/bmo-deployment/overlays/release-0.10"
```

Comment:
These doesn't seem to be used anywhere (neither the newly added or the older). We should clean them up, but let's do it in a separate PR.

Reply:
we will use them in clusterctl upgrade tests, There is a separate task for updating clusterctl upgrade tests; perhaps we can add them when we update those tests.